src/GeekPlatform/Views/KurzusAdatok/Passziv.cshtml

```diff
@@ -1,38 +1,38 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
-*@
-@{
+﻿@{
 }
+@using GeekPlatform.ViewModels.KurzusAdatok;
 
-<div class="row passziv" >
+<div class="row passziv">
     <div class="col-md-8">
-        <h2 id="kurzuscim-passziv">Tárgy neve</h2>
+        <h2 id="kurzuscim-passziv">@Model.Nev</h2>
         <div class="row dobozbal">
             <h4 class="cimsor">Oktató</h4>
-            <div class="col-md-2"></div>
-            <div class="col-md-10">
-                <h4><em>Oktató neve</em></h4>
-                <p>pozíciója</p>
-                <p>e-mail cím<p>
-                <p>telefonszáma</p>
-            </div>
+            @foreach (OktatoViewModel oktato in Model.Oktatok)
+            {
+                <div class="col-md-2"></div>
```

Comment:
Amíg nincsenek profilképek bedobhatnánk ide egy [placeholder](https://placehold.it/) képet.
Az oktatók meg szerintem elférnenek egymás mellé is 2-3-an, [vannak bootstrapes flex classok](https://v4-alpha.getbootstrap.com/utilities/flexbox/#wrap), vagy sima css-el is szép lenne, mert így [elég sok az üres tér](http://i.imgur.com/T1GYQr9.png)